pyproject.toml

```
@@ -0,0 +1,102 @@
+[build-system]
+requires = ["setuptools>=80.0"]
+build-backend = "setuptools.build_meta"
+
+
+[tool.setuptools]
+packages = { find = { include = ["gerrychain*"], exclude = ["tests*"] } }
+
+[project]
+name = "gerrychain"
+version = "1.0.0a2"
+description = "GerryChain is a Python library for building ensembles of districting plans using Markov Chain Monte Carlo methods."
+readme = "README.rst"
+authors = [
+  { name = "MGGG Redistricting Lab", email = "code@mggg.org" },
```

Let's change to Data and Democracy Lab
```suggestion
  { name = "Data and Democracy Lab", email = "code@mggg.org" },
```